packages/shared/lib/core/wallet/actions/events-handlers/handleTransactionInclusionEvent.ts

```diff
@@ -58,8 +58,7 @@ export function handleTransactionInclusionEventInternal(
 
     if (activity?.type === ActivityType.Nft) {
         const isSpendable =
-            (activity.direction === ActivityDirection.Incoming ||
-                activity.direction === ActivityDirection.SelfTransaction) &&
+            (activity.direction === ActivityDirection.Incoming || activity.direction === ActivityDirection.Outgoing) &&
```

Comment:
I dont think this logic is correct, why would an NFT be spendable by you if you send it out?